contribs/gnodev/cmd/gnodev/txs.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/gnolang/gno/tm2/pkg/std"
+)
+
+func loadTxs(txFile string) ([]std.Tx, error) {
```

Comment:
I think `ParseTxs` should be a better name here?

Reply:
done